docs/onboarding/Configuration-Verification-Providers.md

```diff
@@ -67,6 +68,18 @@ innovatrics:
 
 The format of the document name is `{country}_{type}_{edition}` according to the response of `/metadata` request.
 
+
+### Microblink
+
+The BlinkID component from Microblink is used for document verification. The official documentation can be found at [this link](https://blinkidverify.docs.microblink.com/docs/docver/overview/intro).
```

Comment:
🤔 Sure, it is working but could we figure out something better than _this link_?
```suggestion
The BlinkID component from Microblink is used for document verification. For details, see [the official Microblink documentation](https://blinkidverify.docs.microblink.com/docs/docver/overview/intro).
```